### PR TITLE
Update example config's MotD to our current MotD

### PR DESCRIPTION
**Pull request by nervere**
## Why it's good
This doesn't need to be done regularly, by any means, but the current one is literally a decade out of date and still says Baystation.

### CODE REVIEW

**Comment by jwhitak** [comment]
I saw the old motd so many times this week while testing, and was so tempted to do this. Thank you!

**Comment by SonixApache** [comment]
this kills the soul

File: config-example/motd.txt [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
+<b><font size = "4">The current thread can be found by <font color="purple"><a href="https://boards.4channel.org/vg/catalog#s=ss13g">(clicking here)</a></font></font><br></b>
+<font size = "4">Server's byond version is: <a href="https://www.byond.com/docs/notes/515.html">515.1603</a></font>
+<font size = "4">Read the <a href="http://ss13.moe/wiki/">wiki,</a> and consider <a href="http://ss13.moe/wiki/index.php/Guide_to_Contributing_to_the_Wiki">contributing!</a></font><br>
+<font color="#DF0101" size=5>Donate to help keep the server running: <a href="http://ss13.moe/donate">Paypal</a> or <a href="http://patreon.com/pomf">Patreon</a>.</font><br>
+-------- Info --------
+• <b>New player resources:</b> <a href="http://ss13.moe/wiki/index.php/The_Basics">The Basics</a> | <a href="http://ss13.moe/sekrit/rurus.html">Server Rules</a> | <a href="http://ss13.moe/wiki/index.php/Guides">Guides</a>
+• This server's code is <b>open-source</b>, running the Bleeding-Edge branch of /vg/ code. <a href="https://github.com/d3athrow/vgstation13">(Get it here)</a>
+• You can join our official Discord server <a href="https://discord.gg/RYnQZvxrmH">here.</a>
```
Comment (pedr69) [inline config-example/motd.txt:9]: fuck off

Comment (nervere) [reply config-example/motd.txt:9]: Great feedback, thank you!

**Comment by nervere** [comment]
> this kills the soul

There is nothing "soulful" about an unmaintained codebase.

**Comment by SonixApache** [comment]
> > this kills the soul
> 
> There is nothing "soulful" about an unmaintained codebase.

there is a lot soulful about an unmaintained codebase, actually

**Comment by xXxEMBERGHOSTSQUADDIExXx** [comment]
remove discord

**Comment by xXxEMBERGHOSTSQUADDIExXx** [comment]
and add forum link

**Comment by nervere** [comment]
Forums link would be redundant, that's handled separately via the config and is accessible by a button in the top right.

**Comment by Kurfursten** [comment]
I think there is a real objection about startup text inflation. How does this look on a standard window size?

**Comment by nervere** [comment]
> I think there is a real objection about startup text inflation. How does this look on a standard window size?

Here's a screenshot from the live server, it's the exact same as what's in the PR.

image

**Comment by Exxion** [comment]
The version in this PR seems to be slightly older than the version on live. Different BYOND build and different Pomf Facts. Also, I don't think the latter really needs to be in there.

**Comment by Kurfursten** [comment]
Okay, size isn't too bad

**Comment by nervere** [comment]
> The version in this PR seems to be slightly older than the version on live. Different BYOND build and different Pomf Facts. Also, I don't think the latter really needs to be in there.

Removed Pomf facts and fixed the version.

**Comment by Eneocho** [comment]
Pomf facts bros we got too cocky...

**Comment by Eneocho** [comment]
Pomf facts bros we got too cocky...

**Pull request merged**